index.js

```diff
@@ -73,7 +73,7 @@ const updateCommentUrlFromLog = (os,log, comment) => {
   }
   const deployUrl = matches[1]
   application.log(`Deployed the output for ${os} to: ${deployUrl}`)
-  comment.body = comment.body.replace(new RegExp(`- .+? \\(${os}\\)`), `- ${deployUrl} (${os})`)
+  comment.body = comment.body.replace(new RegExp(`- ${os}: .+?`), `- ${os}: ${deployUrl}`)
```

Comment:
You'll need to change the case of the content of the variable `os` as well:
```suggestion
  comment.body = comment.body.replace(new RegExp(`- ${os}: .+?`, "i"), `- ${os.toUpperCase()}: ${deployUrl}`)
```